### PR TITLE
 #909 Bumped up year

### PR DESCRIPTION
**Pull request by dmzaytsev**
for #909 
* bumped up year to 2016
* created a puzzle to update jcabi-parent to latest version

### CODE REVIEW

**Comment by karato** [comment]
@dmzaytsev Thanks for the pull request, let me find a reviewer..

**Comment by karato** [comment]
@carlosmiranda do this review

File: pom.xml [hunk 1]
```diff
@@ -45,6 +45,10 @@
     xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
+        <!--@todo #909:30min jcabi-parent used by Netbout is pretty outdated,
```
Comment (carlosmiranda) [inline pom.xml]: @dmzaytsev This doesn't fit into the ticket description, better to issue a new ticket against it (and get paid separately ;) )

Comment (carlosmiranda) [inline pom.xml]: @dmzaytsev No, hang on, ignore me, it is part of the original ticket. Weird though, I think it should be separated.

**Comment by carlosmiranda** [comment]
@rultor merge please

**Comment by dmzaytsev** [comment]
@karato please deploy

**Comment by karato** [comment]
> @karato please deploy

@dmzaytsev you should ask @rultor about it :)

**Comment by karato** [comment]
@ypshenychka please, review this ticket for compliance with our [QA rules](http://at.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor deploy

**Comment by ypshenychka** [comment]
@carlosmiranda 
According to our [QA Rules](http://www.teamed.io/qa.html):
>The code reviewer found at least three problems in the code.

No issues were found during code review. 
Please confirm that you'll try to find at least three problems while future reviews.

**Comment by carlosmiranda** [comment]
@ypshenychka confirmed, thanks

**Comment by ypshenychka** [comment]
@carlosmiranda Thanks

**Comment by ypshenychka** [comment]
@karato Quality is acceptable.

**Comment by karato** [comment]
> @karato Quality is acceptable.

@ypshenychka thanks for the QA review, we'll work better next time

**Comment by karato** [comment]
@carlosmiranda just added 10 mins to @ypshenychka (for QA), payment ID is `74966775`; Many thanks! **19 mins** were added to your account in Transaction ID `74967020` (task took 14 hours and 24 mins); review comments (c=4) added as a bonus; added +19 to your rating, now it is equal to [+2567](http://www.netbout.com/b/35023?open=rating)

**Pull request merged**